example_projects/transfer/offload_oltp/README.md

```diff
@@ -0,0 +1,197 @@
+### Off-load exist OLTP load into Clickhouse for analytics
+
+Let’s say we have a typical web application that lives inside terraform and consist main storages (let it be Postgres).
+
+And what we need to add some analytical capabilities here:
+
+1. Offload oltp-db analytical to different storage
+2. Aggregate all data in same place
+3. Join it with data outside our scope.
+
+![offload.drawio.svg](offload.drawio.svg)
+
+First of all let’s take a look how to organize code between stages, I do prefer a module + several root here, so we can tweak it a bit easier.
+
+Let’s start with a [`m](http://main.tf)ain.tf`, usually it contains providers definition, nothing more:
+
+```hcl
+provider "doublecloud" {
+  endpoint       = "api.double.cloud:443"
+  authorized_key = file(var.dc-token)
+}
+provider "aws" {
+  profile = var.profile
+}
+```
+
+This will just enable you usage for certain envs, like here it’s AWS and  [Double.Cloud](https://double.cloud/).
+
+First thing we need to do - is create our storage, for this example we will use [Clickhouse](https://clickhouse.com/).
+
+To enable clickhouse we need to create a network were to put it. For this example I choose [BYOA](https://double.cloud/blog/posts/2022/12/bring-your-own-account/) network, so it can be easily peered with exist infra.
+
+```hcl
+module "byoc" {
+  source  = "doublecloud/doublecloud-byoc/aws"
+  version = "1.0.3"
+
+  doublecloud_controlplane_account_id = data.aws_caller_identity.self.account_id
```

Comment:
according the [documentation](https://registry.terraform.io/modules/doublecloud/doublecloud-byoc/aws/latest?tab=inputs) you don't need to set `doublecloud_controlplane_account_id`